internal/kafka/command_topic.go

```diff
@@ -15,7 +15,11 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/log"
 )
 
-const numPartitionsKey = "num.partitions"
+const (
+	// EOF Unicode encoding for Ctrl+D (^D) character
+	EOF              = "\u0004"
```

Comment:
Based on your description, it doesn't matter what this value is equal to. Let's just make it "EOF"?

Reply:
If I make it "EOF", typing "EOF" into `stdIn` will `break` out of the loop, since we're checking the value of `data`. I actually tried exactly this haha!

Reply:
Ah, got it. Does pressing Ctrl-C still print "^C" to the terminal?

Reply:
yup!